python/lsst/qserv/tests/unittest/testIntegration.py

```diff
@@ -48,6 +48,8 @@
 # -----------------------
 class TestIntegration(unittest.TestCase):
 
+    runMulti = None
```

Comment:
I think False would be better here (I traced it to the code in dbLoader where we have "if multi_node:")